pkg/controller/psbackup/controller.go

```diff
@@ -402,6 +401,7 @@ func (r *PerconaServerMySQLBackupReconciler) backupConfig(ctx context.Context, c
 		if !ok {
 			return nil, errors.Errorf("no credentials for Azure in secret %s", nn.Name)
 		}
+		conf.S3.Bucket, _, _ = strings.Cut(s3.Bucket, "/")
 		conf.S3.Bucket = s3.Bucket
```

Comment:
aren't we overwriting the value above with this line?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/258/commits/c43f0b333323600668b3253d7f9fb6ff3eb1146c